recipes/mimalloc/all/conanfile.py

```diff
@@ -142,6 +143,7 @@ def package(self):
             cmake.install()
 
         tools.rmdir(os.path.join(self.package_folder, "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
```

Comment:
Looks like cmake names are not set correctly https://github.com/microsoft/mimalloc/blob/0560fc27c08d28d523b7f741a42deb26cd01c0c6/CMakeLists.txt#L332

Reply:
I'm sorry. I don't know what to do.
If you don't mind, could you tell me what to fix?

Reply:
Whenever I see changes about the CMake config files, my reaction it to look at the changes in the upstream `CMakeLists.txt` ... they fixed up the install dirs but also installed the targets 

You need to configure the names in `package_info` to match

```py
        self.cpp_info.names["cmake_find_package"] = "mimalloc" if self.options.shared else "mimalloc-static"
        self.cpp_info.names["cmake_find_package_multi"] = "mimalloc" if self.options.shared else "mimalloc-static"
```

Reply:
Thank you very much!
Now I understand your point well.

I will remember your point when I update some of the recipes.
My understanding of conan is improved one more.

Reply:
Fixed